VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -337,6 +337,12 @@ def get_total_used_size(self):
                 fstype = ''
                 isNetworkFs = False
                 isKnownFs = False
+
+                if int(used) < 0 :
+                    self.logger.log("The used space is negative, so marking the size computation as failed and returning zero")
+                    self.size_calc_failed = True
+                    return 0,self.size_calc_failed
```

Comment:
Nit: Please use whitespace after the comma ([Reference](https://peps.python.org/pep-0008/#:~:text=Immediately%20before%20a%20comma%2C%20semicolon%2C%20or%20colon%3A))